src/commcare_cloud/ansible/roles/formplayer/defaults/main.yml

```diff
@@ -3,4 +3,9 @@ formplayer_sentry_dsn: ''
 # (you can't use s for seconds, but m, h, d work for minute, hour, day)
 # formplayer_archive_time_spec: '10m'
 formplayer_purge_time_spec: '2d'
-formplayer_release_dir: '{{ formplayer_build_dir }}/{{ "releases/" + new_release_name if new_release_name else "current" }}'
+# setting `this_ansible_run_should_update_formplayer_in_place` to `yes`
+# will result in formplayer updating in place instead of through a release / symlink sequence.
+# Unless tasks are further filtered (i.e. with a tag), this can cause unncessary downtime.
+# This variable should never be set in public.yml,
+# and is only meant to be used by update-config
+this_ansible_run_should_update_formplayer_in_place: no
```

Comment:
This long name makes some lines where it is referenced quite long. Could rename to `_should_update_formplayer_in_place` without losing any information. I assume ansible allows underscore-prefixed names, which seems like a reasonable indication that it's private.